src/lib/autocomplete/OVERVIEW.md

```diff
@@ -0,0 +1,149 @@
+
+The autocomplete is a normal text input enhanced by a panel of suggested options. You can read more about 
+autocompletes in the [Material Design spec](https://material.io/guidelines/components/text-fields.html#text-fields-auto-complete-text-field).
+
+### Simple autocomplete
+
+Start by adding a regular `mdInput` to the page. Let's assume you're using the `formControl` directive from the 
+`@angular/forms` module to track the value of the input.
+
+*my-comp.html*
+```html
+<md-input-container>
+   <input type="text" mdInput [formControl]="myControl">
+</md-input-container>
+```
+
+Next, create the autocomplete panel and the options displayed inside it. Each option should be defined by an 
+`md-option` tag. Set each option's value property to whatever you'd like the value of the text input to be 
+upon that option's selection.
+ 
+*my-comp.html*
+```html
+<md-autocomplete>
+   <md-option *ngFor="let option of options" [value]="option">
+      {{ option }}
+   </md-option>
+</md-autocomplete>
+```
+
+Now we'll need to link the text input to its panel. We can do this by exporting the autocomplete panel instance into a 
+local template variable (here we called it "auto"), and binding that variable to the input's `mdAutocomplete` property.
+
+*my-comp.html*
+```html
+<md-input-container>
+   <input type="text" mdInput [formControl]="myControl" [mdAutocomplete]="auto">
+</md-input-container>
+
+<md-autocomplete #auto="mdAutocomplete">
+   <md-option *ngFor="let option of options" [value]="option">
+      {{ option }}
+   </md-option>
+</md-autocomplete>
+```
+
+### Adding a custom filter
+
+At this point, the autocomplete panel should be toggleable on focus and options should be selectable. But if we want 
+our options to filter when we type, we need to add a custom filter. 
+
+We already have access to the built-in `valueChanges` observable on the `FormControl`, so we can simply map the text 
```

Comment:
This feels a bit like you're saying "This is the One True Way to do filtering". What do you think of approaching more as "You can filter the options in any way you want based on the text input. Here's one example."